fixtures/ed/ntfs/admin_stations.txt

```diff
@@ -0,0 +1,2 @@
+admin_id,admin_name,station_id,station_name
+admin:A,"admin_name_A_that_is_no_read",A,"station_name_A_that_is_no_read"
```

Comment:
:+1: and this test will eventually be removed when navitia stops reading station_id

Reply:
Yes indeed